From 2dfb0eb6c296e1d77a4d6fbd8ee7ce50a2379825 Mon Sep 17 00:00:00 2001
From: Ian Lance Taylor <iant@golang.org>
Date: Thu, 20 Aug 2015 21:20:25 -0700
Subject: [PATCH 06/63] [release-branch.go1.5] cmd/go: -a does apply to the
 standard library

This changed in https://golang.org/cl/10761.

Update #12203.

Change-Id: Ia37ebb7ecba689ad3cb2559213d675f21cf03a95
Reviewed-on: https://go-review.googlesource.com/13799
Reviewed-by: Russ Cox <rsc@golang.org>
Reviewed-on: https://go-review.googlesource.com/14226
Reviewed-by: Minux Ma <minux@golang.org>
---
 src/cmd/go/alldocs.go | 1 -
 src/cmd/go/build.go   | 1 -
 2 files changed, 2 deletions(-)

diff --git a/src/cmd/go/alldocs.go b/src/cmd/go/alldocs.go
index 6077d93..1134997 100644
--- a/src/cmd/go/alldocs.go
+++ b/src/cmd/go/alldocs.go
@@ -81,7 +81,6 @@ and test commands:
 
 	-a
 		force rebuilding of packages that are already up-to-date.
-		In Go releases, does not apply to the standard library.
 	-n
 		print the commands but do not run them.
 	-p n
diff --git a/src/cmd/go/build.go b/src/cmd/go/build.go
index b7c7e05..0b14725 100644
--- a/src/cmd/go/build.go
+++ b/src/cmd/go/build.go
@@ -60,7 +60,6 @@ and test commands:
 
 	-a
 		force rebuilding of packages that are already up-to-date.
-		In Go releases, does not apply to the standard library.
 	-n
 		print the commands but do not run them.
 	-p n
-- 
2.6.1

